### PR TITLE
Passing Along My Quiz

### PR DESCRIPTION
**Pull request by nathantbaker**

### CODE REVIEW

File: JS/1-CarLot.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  return {
+    // Pull data from JSON file
+    loadInventory: function (callback) {
+      var loader = new XMLHttpRequest();
+      loader.addEventListener("load", function () {
+        _cars = JSON.parse(this.responseText).cars;
+        callback(_cars);
+      });
+
+      loader.open("GET", "../JSON/inventory.json");
```
Comment (gregkorte) [inline JS/1-CarLot.js]: This file reference is backing up one too many directories and throwing an error.

File: JS/2-eventHandlers.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var CarLot = (function (oldCarLot) {
+
+  CarLot.activateEvents = function() {
```
Comment (gregkorte) [inline JS/2-eventHandlers.js]: This function declaration should be prefixed with what you have passed into the IIFE and not the IIFE itself.

File: JS/3-DOM.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+var CarLot = (function (oldCarLot) {
+
+  CarLot.populatePage = function (inventory) {
```
Comment (gregkorte) [inline JS/3-DOM.js]: This is another one of those prefix issues.

File: JS/3-DOM.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  html += rowEnd;
+  let target = document.getElementsByClassName("container")[1];
+  target.innerHTML += html; // push string to page
+
+  // Create listeners after DOM is populated
+  CarLot.activateEvents();
+
+  };
+
+  CarLot.resetCars = function () {
```
Comment (gregkorte) [inline JS/3-DOM.js]: Prefix.

File: JS/3-DOM.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    let target = document.getElementsByClassName("car");
+    for (var i = 0; i < target.length; i++) {
+      target[i].classList.remove("border"); // remove border class from car cards
+      target[i].style.backgroundColor = "white"; // set all car cards to white.
+    }
+  };
+
+  CarLot.highlightCar = function (element, newColor) {
```
Comment (gregkorte) [inline JS/3-DOM.js]: Prefix.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  </head>
+  <body>
+
+    <div class="navbar">
+      <div class="navbar-inner">
+        <div class="container">
+
+          <a class="brand" href="#"><h1>Nate's Car Lot</h1></a>
+
+          <form class="navbar-form">
```
Comment (gregkorte) [inline index.html]: Forms will always refresh your page, so when I hit my `Enter` key the page refreshes, and my changes are gone.

**Comment by gregkorte** [comment]
Nice work on your quiz, Nate! Easy to follow, succinct code. Please see the few comments I've made, and let me know when you're ready for me to take another look.

**Comment by nathantbaker** [comment]
@gregkorte thanks for the feedback! I made changes based on your recommendations.

**Comment by gregkorte** [comment]
Looks good, Nate! You're good to merge! :+1:

**Comment by nathantbaker** [comment]
🌮  🎉

**Pull request merged**